dotnet/src/Planners/Planners.Core/Sequential/SequentialPlannerConfig.cs

```diff
@@ -13,9 +13,12 @@ namespace Microsoft.SemanticKernel.Planners;
 public sealed class SequentialPlannerConfig : PlannerConfigBase
 {
     /// <summary>
-    /// The maximum number of tokens to allow in a plan.
+    /// Initializes a new instance of the <see cref="SequentialPlannerConfig"/> class.
     /// </summary>
-    public int? MaxTokens { get; set; }
+    public SequentialPlannerConfig()
+    {
+        this.MaxTokens = 1024;
```

Comment:
I'm tempted to have these all move to 4000 to support `gpt-35-turbo` by default. Thoughts?

Reply:
This would improve consistency between planners so I would be in favor of this.

Reply:
Is there a reason why the Action and Sequential planners have lower token limits currently?

Reply:
Mostly an artifact of when `text-davinci-003` was the low end.

Reply:
Yeah, we should update to the max and as part of this PR make the Action and Sequential Planner do token counting to make sure developers hit token-limit-errors less frequently.

Reply:
Okay I forgot we'd need to remove the dependency on semantic functions so maybe best to keep that separate. Sorry for the distraction and thanks for highlighting this.